source/components/patternfly/tests-src/_includes/widgets/communication/alert-danger-dismissable.html

```diff
@@ -0,0 +1,7 @@
+<div class="alert alert-danger alert-dismissable">
```

Comment:
You may have had the soft link in a previous commit -- most likely how they got in? Regardless, you won't find source/components in patternfly-toolkit if you clone a new repo.

Try deleting the files to see if it will solve the problem. Although, we want to leave the original soft link in place. If it doesn't work,let me know. You can always copy the modified files to a new, clean branch.